packages/server-core/src/networking/instance/instance.hooks.ts

```diff
@@ -140,25 +140,25 @@ export default {
   before: {
     all: [() => schemaHooks.validateQuery(instanceQueryValidator), schemaHooks.resolveQuery(instanceQueryResolver)],
     find: [
-      iff(isProvider('external') && isAction('admin'), verifyScope('admin', 'admin'), addLocationSearchToQuery),
+      iff(isProvider('external') && isAction('admin'), verifyScope('instance', 'read'), addLocationSearchToQuery),
       discardQuery('search'),
       discardQuery('action'),
       sortByLocationName
     ],
     get: [],
     create: [
-      iff(isProvider('external'), verifyScope('admin', 'admin')),
+      iff(isProvider('external'), verifyScope('instance', 'write')),
       () => schemaHooks.validateData(instanceDataValidator),
       schemaHooks.resolveData(instanceDataResolver),
       addRoomCode
     ],
-    update: [iff(isProvider('external'), verifyScope('admin', 'admin'))],
+    update: [],
```

Comment:
This should be disallowed. We don't call this anywhere, but if we don't plan to control what happens when update is called, then we should just not let anyone call it at all.